ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/config-comment.yml

```
@@ -0,0 +1,3 @@
+config_comment: |
```

Your check for blank lines hides the fact that you are inserting a closing newline with the current input. By adding `-` you avoid that.
```suggestion
config_comment: |-
```

---

Done!